docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.9"
+services:
+  app:
+    command: /bin/bash
+    build: .
+    volumes: 
+      - ../pacta-data:/pacta-data
```

Comment:
Suggestion: when in active development, I do find it useful to mount the entire project (or at least the code) directory as a volume, so that I can edit in my native (host) environment, and have those changes picked up instantly by the container. Helps reduce time when iterating, it requires running with `docker-compose -f docker-compose.yml up`, after there's a candidate, but at least you don't have to do that every time you make a little change.